client/src/scenes/battle_select_scene.rs

```diff
@@ -74,6 +76,13 @@ impl BattleSelectScene {
             .package(PackageNamespace::Local, package_id)
             .unwrap();
 
+        let package_name = package.name.as_str().to_uppercase();
```

Comment:
No need to use `as_str()`, every `str` method is available on `String`. My understanding is this works from String implementing the Borrow<str> trait, it's a bit hidden if you only look at the documentation.